main.py

```diff
@@ -1,13 +1,45 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, HTTPException
 import uvicorn
-
+from pydantic import BaseModel
 from router import router
+from typing import List
 
 
 app = FastAPI()
 
 app.include_router(router)
 
 
+class User(BaseModel):
+    first_name: str
+    last_name: str
+    age: int
+
+
+user_table = [
+    User(first_name="user1", last_name="last_name1", age=2135),
+    User(first_name="user2", last_name="last_name1", age=55),
+    User(first_name="user3", last_name="lastname_666", age=20),
+]
+
+
+@app.get("/all", response_model=List[User])
+async def get_all_users():
+    return user_table
+
+# 2. Эндпоинт для создания нового пользователя
+@app.post("/create", response_model=User)
+async def create_user(user: User):
+    user_table.append(user)
+    return user
+
+# 3. Эндпоинт для поиска пользователя по имени
+@app.get("/get_by_name", response_model=List[User])
```

Comment:
тут лучше использовать втроенный в питне тип - list
```suggestion
@app.get("/get_by_name", response_model=list[User])
```

Reply:
плюс возвращаемый тип